nextcord/http.py

```diff
@@ -1862,6 +1862,21 @@ def ban(
             retry_request=retry_request,
         )
 
+    def bulk_ban(
+        self,
+        guild_id: Snowflake,
+        user_ids: List[Snowflake],
```

Comment:
The positional args for .ban are  `user_id, guild_id` but .bulk_ban has them reversed (`guild_id, user_ids`)? Maybe I'm being nitpicky, but it feels inconsistent?

Reply:
Good point, I (most likely) didn't notice it when the code